docs/source/clickhouse/topology.rst

```diff
@@ -0,0 +1,45 @@
+==================================
+ClickHouse Topology Best Practices
+==================================
+
+Sentry has a few conventions for using ClickHouse that are not inherent to
+the database but useful to consider if you intend to create new tables or
+datasets.
+
+Storage nodes vs. Query nodes
```

Comment:
I had thought there was a general idea to move away from separate query nodes and put the distributed table directly on the storage nodes, like metrics. Or was that specific to metrics?

Reply:
That was specific to release-health metrics (dataset `metrics`) and we're going to go back and add query nodes to those because it's actually been problematic for ops already to not have that layer of abstraction